README.md

```diff
@@ -1,4 +1,34 @@
+## Getting Started
+### Example App
```

Comment:
`Example App` and `Register Your App` paragraphs should not be visible on this repo. This repo should have only instruction for use the SDK not for setup an app. Setup an app is specific to SC2 and not specific to SC2-SDK JS, there can be multiple SDK (PHP, Python etc...) we will not duplicate these informations everywhere.

Reply:
Updated just to add a link for general information on SC2 and setting up an app. I think it's important that if someone goes here first they at least have a link to get more info.